docs/CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,179 @@
+
+Committers have all the rights and responsibilities of a reviewer, plus:
+
+* Responsibilities include:
+    * Reviewing most Pull Requests against their specific areas of responsibility
+    * 5 accepted PRs with code changes (not just docs or typo fixes) per year
+    * Helping other contributors become Committers
+* Requirements:
+    * Experience as a Reviewer for at least 3 months
+    * Has reviewed, or helped review, at least 15 Pull Requests
+    * Has analyzed and resolved test failures in their specific area
+    * Has demonstrated an in-depth knowledge of the specific area
+    * Commits to being responsible for that specific area
+    * Is supportive of new and occasional contributors and helps get useful PRs in shape to commit
+* Additional privileges:
+    * Has GitHub or CI/CD rights to approve pull requests in specific directories
+    * Can recommend and review other contributors to become Committers
+    * Is listed as Committers in the [OWNERS file](./OWNER.md) under the WasmEdge/WasmEdge repos.
+  
+
+The process of becoming a Committer is:
+
+1. The contributor is nominated by opening a PR against the appropriate repository by the existing reviewers, committers and maintainers, which adds their GitHub username to the OWNERS file for one or more directories.
+2. At least 1/2 members of the reviewers, committers, and maintainers that owns that repository or main directory, who are already Approvers, approve the PR.
+
+
+### Maintainer
+<!-- In the simplest and most common project structures, projects have a single pool of "maintainers" who are collectively responsible for the entire project.  This example defines that role for your project.  If your project has a more complex structure, see the list of specific maintainer roles you might want to define, below. -->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+
+Description: Maintainers are very established contributors who are responsible for the entire project. As such, they have the ability to approve PRs against any area of the project, and are expected to participate in making decisions about the strategy and priorities of the project.
+
+A Maintainer must meet the responsibilities and requirements of a Reviewer, plus:
+
+* Responsibilities include:
+    * Reviewing at least 30 PRs per year, especially PRs that involve multiple parts of the project
+    * Mentoring new Reviewers
+    * Writing refactoring PRs
+    * Participating in CNCF maintainer activities
+    * Determining strategy and policy for the project
+    * Participating in, and leading, community meetings
+* Requirements
+    * Experience as a Reviewer for at least 3 months
+    * Demonstrates a broad knowledge of the project across multiple areas
+    * Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+    * Mentors other contributors
```

Comment:
Is this a requirement or a responsibility?

Reply:
Let me check

Reply:
It's a requirement.